homeassistant/components/guardian/__init__.py

```diff
@@ -102,7 +102,7 @@ async def disable_ap(call):
         """Disable the device's onboard access point."""
         try:
             async with guardian.client:
-                await guardian.client.device.wifi_disable_ap()
+                await guardian.client.wifi.disable_ap()
```

Comment:
All these services should be moved to the switch platform and use `entity_platform.async_register_entity_service(…)`.

With the current setup we can only handle 1 guardian device or else the services will override one another.

Reply:
(in a different PR)